RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,24 +8,39 @@
 
 import Cocoa
 
+protocol EntitiesViewDataSource : NSObjectProtocol {
+    func numberOfRowsInEntitiesView(entitiesView:EntitiesView) -> Int
+    func entitiesView(tableView:NSTableView, viewForTableColmn tableColumn: NSTableColumn?, row: Int) -> NSView?
+}
+
+protocol EntitiesViewDelegate : NSObjectProtocol {
+    func addEntityInEntitiesView(entitiesView:EntitiesView)
+    func removeEntityInEntitiesView(entitiesView:EntitiesView, index:Int)
+    func entitiesView(entitiesView:EntitiesView, selectionDidChange index:Int)
+}
+
 @IBDesignable
 class EntitiesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSource {
     
     @IBOutlet var tableView:NSTableView!
-    
     @IBOutlet var addButton:NSButton!
     @IBOutlet var removeButton:NSButton!
     
-    override func nibDidLoad() {
-        super.nibDidLoad()
-    }
+    var dataSource:EntitiesViewDataSource?
```

Comment:
make dataSource and delegate weak types, i can talk about this more tomorrow but feel free to google why